### PR TITLE
CBL-3628 Maui test project to include Android platform

### PR DESCRIPTION
**Pull request by Sandychuang8**
All the changes in this PR is to make the Maui android test project build and able to use CBL libraries.
Some methods use in the tests are disabled due to incompatibility with .Net 6 Android/Maui Android and it requires time to research and investigate.
[CBL-3711](https://issues.couchbase.com/browse/CBL-3711) is created for the job.
And there also some indentation update in this PR.

### CODE REVIEW

File: src/Couchbase.Lite.Shared/Sync/HTTPLogic.cs [hunk 1]
```diff
@@ -195,9 +195,9 @@ private static string GetUserAgent()
 			var version = versionAtt?.InformationalVersion ?? "Unknown";
             var regex = new Regex("([0-9]+\\.[0-9]+\\.[0-9]+)-b([0-9]+)");
 			var build = "0";
-            var commit = ThisAssembly.Git.Commit;
+            //var commit = ThisAssembly.Git.Commit;
```
Comment (borrrden) [inline src/Couchbase.Lite.Shared/Sync/HTTPLogic.cs:198]: Why?

File: src/Couchbase.Lite.Support.Android/Activate.cs [hunk 2]
```diff
@@ -56,7 +56,7 @@ public static void Activate(Context context)
             Context = context ?? throw new ArgumentNullException(nameof(context));
         }
 
-        internal static void CheckVersion()
+        public static void CheckVersion()
```
Comment (borrrden) [inline src/Couchbase.Lite.Support.Android/Activate.cs:59]: This is not the way to handle this, did you use the `InternalsVisibleTo` attribute?

File: src/Couchbase.Lite.Support.Android/Couchbase.Lite.Support.Android.csproj [hunk 3]
```diff
@@ -47,13 +47,21 @@
...
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(TargetFramework)' == 'Debug|net6.0-android31' ">
+    <DefineConstants>$(DefineConstants);NET6_0_ANDROID</DefineConstants>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(TargetFramework)' == 'Release|net6.0-android31' ">
+    <DefineConstants>$(DefineConstants);NET6_0_ANDROID</DefineConstants>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)' == 'Packaging|net6.0-android31'">
+    <DefineConstants>$(DefineConstants);NET6_0_ANDROID</DefineConstants>
+  </PropertyGroup>
```
Comment (borrrden) [inline src/Couchbase.Lite.Support.Android/Couchbase.Lite.Support.Android.csproj:64]: This is repetitive.  Why not just use targetframework?

**Review by borrrden — CHANGES_REQUESTED** [review]
Quite hard to tell what all these changes are for, but a few comments

File: src/Couchbase.Lite.Shared/Sync/HTTPLogic.cs (hunk 1 above)
Comment (Sandychuang8) [reply src/Couchbase.Lite.Shared/Sync/HTTPLogic.cs:198]: strange, I thought I have already uncommented these...

**Review by borrrden — APPROVED** [review]

**Pull request merged**